packages/app-check-compat/src/service.test.ts

```diff
@@ -65,7 +65,7 @@ describe('Firebase App Check > Service', () => {
 
   it(
     'activate("string") calls modular initializeAppCheck() with a ' +
-    'ReCaptchaV3Provider',
+      'ReCaptchaV3Provider',
```

Comment:
is it correctly formatted?

Reply:
Yes, this seems to be prettier's rule. It's the same through the whole file.